okcupyd/profile.py

```diff
@@ -161,8 +161,10 @@ def liked(self):
         classes = self._liked_xpb.one_(self.profile_tree).attrib['class'].split()
         return 'liked' in classes
 
-    _contacted_xpb = xpb.div(id='actions').div.with_classes(
-        'tooltip_text', 'hidden'
+    _contacted_xpb = xpb.div.with_class('actions2015').button.attribute_contains(
+        'class','actions2015-chat flatbutton blue'
```

Comment:
performed stylistic edits and improved xpb precision in https://github.com/kfred/okcupyd/commit/da49f95d1fdeddd57f96d82ced2050cd32c29bcb